magefiles/adr/magefile.go

```diff
@@ -0,0 +1,22 @@
+package adr
+
+import (
+	"github.com/magefile/mage/sh"
+)
+
+// Install installs adr tools in a specified version
+func Install() error {
+	return sh.RunV(
```

Comment:
Also check if not already installed

Reply:
In that case, the command will not update the tool if the old version is installed.

We could alternatively supply version as an argument (that defaults to `latest` if it is possible) and check if installed, if version matches, and the install/update the tool.

Reply:
it gots releases, you could basically download the binary as well.... 🤔 We've got functions for that. (github to get latest relase and http to download. check if the executable is in path and what version, if same, then skip. 

Although I don't suppose it'll change much since last release was in 2018 🤔

Reply:
Done